src/components/TokenPairSelector/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { TokenPairSelectorList } from 'components/TokenPairSelectorList'
+
+import ArrowWhite from 'assets/img/arrow-white.svg'
+
+const TokenPairSelectorStyled = styled.div`
+  display: flex;
+  width: 100%;
+
+  > button {
```

Comment:
I'm wondering. Should we have our own styled buttons instead of doing it here?
same approach as Uniswap.

Reply:
In this case, while using a `<button>` it will not be a button like we will use else where. Just decided that using a `<button>` element might be most appropriate here. So I regard this a specific use case. 

That being said, I think it would be good to have a reusable styled button where applicable.

Reply:
see [my comment here](https://github.com/gnosis/dex-react/issues/1604#issuecomment-725997685) regarding this. It should be a styled button we inherit from for more specific cases. But fine to flesh out as pure CSS now and rework later

Reply:
I can try a start with that. I guess we try to fit all types of button logic into a generic `Button` component? Even if they look/behave entirely different, correct?

Reply:
> I can try a start with that. I guess we try to fit all types of button logic into a generic Button component? Even if they look/behave entirely different, correct?

correct!